.github/workflows/application-signals-java-e2e-ec2-canary-test.yml

```diff
@@ -16,16 +16,30 @@ permissions:
   contents: read
 
 jobs:
-  e2e-test:
+  e2e-ec2-default-test:
     strategy:
       fail-fast: false
       matrix:
         aws-region: ['af-south-1','ap-east-1','ap-northeast-1','ap-northeast-2','ap-northeast-3','ap-south-1','ap-south-2','ap-southeast-1',
                      'ap-southeast-2','ap-southeast-3','ap-southeast-4','ca-central-1','eu-central-1','eu-central-2','eu-north-1',
                      'eu-south-1','eu-south-2','eu-west-1','eu-west-2','eu-west-3','il-central-1','me-central-1','me-south-1', 'sa-east-1',
                      'us-east-1','us-east-2', 'us-west-1', 'us-west-2']
-    uses: ./.github/workflows/application-signals-java-e2e-ec2-test.yml
+    uses: ./.github/workflows/application-signals-java-e2e-ec2-default-test.yml
     secrets: inherit
     with:
       aws-region: ${{ matrix.aws-region }}
-      caller-workflow-name: 'appsignals-e2e-ec2-canary-test'
\ No newline at end of file
+      caller-workflow-name: 'appsignals-e2e-ec2-canary-test'
+
+  e2e-ec2-asg-test:
+    strategy:
+      fail-fast: false
+      matrix:
+        aws-region: ['af-south-1','ap-east-1','ap-northeast-1','ap-northeast-2','ap-northeast-3','ap-south-1','ap-south-2','ap-southeast-1',
+                     'ap-southeast-2','ap-southeast-3','ap-southeast-4','ca-central-1','eu-central-1','eu-central-2','eu-north-1',
+                     'eu-south-1','eu-south-2','eu-west-1','eu-west-2','eu-west-3','il-central-1','me-central-1','me-south-1', 'sa-east-1',
+                     'us-east-1','us-east-2', 'us-west-1', 'us-west-2']
```

Comment:
Can we double check if any new regions have been added with the latest release?

Reply:
Yeah no new regions